### PR TITLE
Create Crystals on Crystal Set Show Page

### PR DESCRIPTION
**Pull request by suzkiee**
# Description
Completes User Story 13.

## Fixes # (issue)
Adds logic to create new crystals from the crystal set show page. 

## Type of change
-[x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Green tests written based off the user stories

### CODE REVIEW

File: app/views/set_crystals/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <a href="/crystal_sets">Back to Crystal Sets </a>
 <a href="/crystals">All Crystals</a>
-<h1><%= @set.collection_name %> Collection </h1>
+<h1><%= @crystal_set.collection_name %> Collection </h1>
```
Comment (LawrenceWhalen) [inline app/views/set_crystals/index.html.erb:3]: Nice find.

File: app/views/set_crystals/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<%= form_with url: "/crystal_sets/#{@crystal_set.id}/crystals", method: :post, local: true do |form| %>
+  <%= form.label :name, "Name" %>
+  <%= form.text_field :name %>
```
Comment (LawrenceWhalen) [inline app/views/set_crystals/new.html.erb:3]: if you add `:required => true' inside the form will make is so the field has to have something in it to pass on.

**Review by LawrenceWhalen — APPROVED** [review]
Commits and code look great.

File: app/views/set_crystals/new.html.erb (hunk 2 above)
Comment (suzkiee) [reply app/views/set_crystals/new.html.erb:3]: 👀 oooh lala!! I will incorporate this

**Pull request merged**